app/models/pusher.rb

```diff
@@ -126,10 +126,21 @@ def after_write
     Indexer.perform_later
     ReindexRubygemJob.perform_later(rubygem:)
     GemCachePurger.call(rubygem.name)
+    StoreVersionContentsJob.perform_later(version:) if ld_variation(key: "gemcutter.pusher.store_version_contents", default: false)
     RackAttackReset.gem_push_backoff(@remote_ip, @user.display_id) if @remote_ip.present?
     StatsD.increment "push.success"
   end
 
+  def ld_context
+    LaunchDarkly::LDContext.with_key(user.display_id, "user")
```

Comment:
nit: let's add an `ld_context` method to the `user` model, and have it return something like:

```ruby
LaunchDarkly::LDContext.create({
    key: "user-id-#{id}",
    kind: "user",
    name:
})
```

just because the `display_id` can change over time, and we want the context key to be stable

Reply:
updated.